### PR TITLE
DOC: Improve docs of brain views

### PR DESCRIPTION
**Pull request by larsoner**
1. Improve enumeration of `views` in brain by describing them
2. Add `RAS` entry to glossary
3. Add links to methods in `Brain` comparison table
4. Remove two methods that we had in the comparison table but didn't actually have implementations for (`remove_foci` and `scale_data_colormap`)

### CODE REVIEW

**Comment by larsoner** [comment]
https://41400-1301584-gh.circle-artifacts.com/0/dev/generated/mne.viz.Brain.html#mne.viz.Brain.show_view

File: mne/viz/_brain/_brain.py [hunk 1]
```diff
@@ -2939,6 +2934,44 @@ def show_view(self, view=None, roll=None, distance=None, *,
...
+
+        ``'lateral'``
+            From the left or right side such that the lateral (outside)
+            surface of the given hemisphere is visible.
+        ``'medial'``
+            From the left or right side such that the medial (inside)
+            surface of the given hemisphere is visible (at least when in split
+            or single-hemi mode).
+        ``'rostral'``
+            From in front.
```
Comment (mmagnuski) [inline mne/viz/_brain/_brain.py:2953]:
```suggestion
            From the front.
```

**Comment by mmagnuski** [comment]
Looks good! 👍

**Comment by mmagnuski** [comment]
@larsoner a side question: Is there a resource where I could check the commit magic commands like `[skip actions]`? I would like to reduce the burden on the CIs when pushing cosmetic commits.

**Comment by GuillaumeFavelier** [comment]
Thank you @larsoner !

**Comment by mmagnuski** [comment]
Thanks @larsoner (also for the link!)

**Pull request merged**